tests/test_component/test_tmgr_base.py

```diff
@@ -9,16 +9,29 @@
 from pika.connection import ConnectionParameters
 
 import pika
+import time
+import psutil
+import threading as mt
+import multiprocessing as mp
 
 try:
     import mock
 except ImportError:
     from unittest import mock
 
 
+# ------------------------------------------------------------------------------
+#
+def _tmgr_side_effect(event):
+
+    while not event.is_set():
+        continue
```

Comment:
you may want to add a small sleep (`0.01 or so`)here as not to have a busy wait loop ...?

Reply:
Thank you!